tests/schema.php

```diff
@@ -0,0 +1,61 @@
+<?php
+declare(strict_types=1);
+
+use Cake\Database\Schema\TableSchema;
+
+/**
+ * Abstract schema for Debugkit tests.
+ */
+return [
+    [
+        'table' => 'panels',
```

Comment:
I wonder if we should also update the schema generation tricks we use in DebugKit to build the database at runtime to also use this schema format.